### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by ghost** [Deltares/GEOLib-Plus#16] [General]
Hi GEOLib team,

I am a regular user of the 'normal' GEOLib package. I am also interested in using the GEOLib+ package. I was wondering, is there a documentation page somewhere? I always have a hard time finding the documentation page of GEOLib, since it does not show up in google searches and it is not linked anywhere. I have bookmarked that page to find it quickly. 

So is there a documentation page for GEOLib+ and it maybe makes sense to link it in the README and or PYPI page.

edit; I see now that de documentation for the regular GEOLib is linked in the PyPi page. (but not in the readme)

Thanks,

Maarten Weehuizen

**Comment by EleniSmyrniou** [comment]
Hello Maarten,

We do have a documentation page but it seems not to be working at the moment.
Thanks for letting us know!

**Comment by EleniSmyrniou** [comment]
Hey Maarten,
In the end we fixed the documentation!
